content-src/components/Sections/Sections.jsx

```diff
@@ -248,11 +248,11 @@ export class Section extends React.PureComponent {
           </div>}
         {id === "topstories" &&
           <div className="top-stories-bottom-container">
-            <div>
+            <div className={`${shouldShowTopics ? `wrapper-topics` : `wrapper-cta`}`}>
```

Comment:
This is causing a linting error because it's pushing the complexity of `render` up too high.

`Method 'render' has a complexity of 35  complexity`

@k88hudson @ScottDowne – How do you normally deal with this? Do you set a linting exception per file? I could externalize this logic, but that feels more confusing and harder to maintain.

Reply:
I hit this issue in this exact file not long ago,I filed it with something like the first one below. (was the easiest at the time)

I bet if if you had a `const sensibleClassName = shouldShowTopics ? 'wrapper-topics' : 'wrapper-cta'` up top, and `className={sensibleClassName}` in the div, it would fix it.

Another solution could probably something like this (they both shouldn't ever be able to be rendered):
```
          <div className="top-stories-bottom-container">
            {shouldShowTopics &&
              <div className="wrapper-topics">
                <Topics topics={this.props.topics} />
              </div>
            }
            {shouldShowPocketCta &&
              <div className="wrapper-cta">
                <PocketLoggedInCta />
              </div>
            }
            <div className="wrapper-more-recommendations">
              {shouldShowReadMore &&
                <MoreRecommendations read_more_endpoint={read_more_endpoint} />}
            </div>
          </div>
```

but maybe we just need a `{this.renderBottomContainer()}`